ivy/functional/frontends/jax/numpy/mathematical_functions.py

```diff
@@ -503,3 +503,8 @@ def conj(x, /):
 def subtract(x1, x2):
     x1, x2 = promote_types_of_jax_inputs(x1, x2)
     return ivy.subtract(x1, x2)
+
+
+@to_ivy_arrays_and_back
+def around(a, decimals=0, out=None):
+    return ivy.around(a, decimals, out)
```

Comment:
I do not think that there is a "ivy.around" function for you to call directly like that. Thank you

Reply:
I have fixed it, please have a look.